kuma/static/styles/components/structure/login.scss

```diff
@@ -16,7 +16,7 @@ Header login link
         vertical-align: top; /* needed on mobile */
     }
 
-    > a:nth-child(n+2) {
+    > a, button:nth-child(n+2) {
```

Comment:
As you have mentioned earlier, need to place the login ``<a>`` before the form. So should we still do that?
@jpetto Can you suggest what will be the selector? I dont have much idea about CSS.